package.json

```diff
@@ -70,9 +70,7 @@
     }
   },
   "files": [
-    "./dist/*",
-    "README.md"
```

Comment:
Why is this not needed? Is this included anyway?

Reply:
Readme is not needed, dist is.